apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.sql

```diff
@@ -1,4 +1,4 @@
--- BLOCK select_lti13_instances
+-- BLOCK select_combined_lti13_instances
```

Comment:
What does "combined" mean here? That's not something I recognize from our patterns.

Reply:
For instance, I discovered I use the types "Lti13FullInstance" in this page for a similar thing. I'll make another PR to fold that into the Lti13CombinedInstance terminology.

Reply:
The convention we've developed in PT is to have a query like `select_lti13_instances`, but name the schema `Lti13InstanceRowSchema`. A "row" can be composed of data from many different tables. We might want to do the same here? cc @mwest1066 in case he wants to add anything to this, he's been the main proponent of this pattern.

Reply:
Similar to in #12009, I'm inclined to make incremental progress with renaming to combined_lti13_instances here, and if we decide to do something like `Lti13CourseInstanceRowSchema` that has `Lti13InstanceSchema` in it, we refactor in a future PR that fixes all the locations.